region/brinstar/blue.json

```diff
@@ -1114,50 +1114,50 @@
                 {
                   "name": "Go Back Up",
                   "notable": false,
-                  "requires": ["Morph"], 
+                  "requires": [
+                    "Morph",
+                      {"or": [
+                        "canWalljump",
+                        "h_canFly"
+                      ]}
+                  ], 
                   "obstacles": [
                     {
                       "id": "B",
                       "requires": ["never"]
                     },
                     {
                       "id": "A",
-                      "requires": [
-                        {"or":[
-                          "ScrewAttack",
-                          "Bombs",
-                          {"and":[
-                            "PowerBomb",
-                            {"ammo": {
-                              "type": "PowerBomb",
-                              "count": 1
-                            }}
-                          ]}
-                        ]}
-                      ]
+                      "requires": [ "h_canDestroyBombWalls" ]
                     }
                   ],
                   "note": "If the Crumble Block has been destroyed while going down, it's possible to go back up without breaking the PB blocks."
                 },
                 {
                   "name": "Break the PB Blocks",
                   "notable": false,
-                  "requires": ["h_canUsePowerBombs"],
+                  "requires": [
+                    "h_canUsePowerBombs",
+                      {"or": [
+                        "canWalljump",
+                        "h_canFly",
+                          {"canShineCharge": {
+                            "usedTiles": 33,
+                            "shinesparkFrames": 43,
```

Comment:
Doesn't this mean that the shinespark approach has to pay twice the energy cost?

It might be preferrable to break this strat in two, so we can have a dedicated shinespark strat that clears the obstacle for free.

Reply:
Good catch. This should fix it.